public/app/core/navigation/GrafanaRouteError.tsx

```diff
@@ -51,7 +49,9 @@ export function GrafanaRouteError({ error, errorInfo }: Props) {
   );
 }
 
-const getStyles = () => css`
-  width: 500px;
-  margin: 64px auto;
-`;
+const getStyles = () => ({
+  container: css({
+    width: '500px',
+    margin: '64px auto',
```

Comment:
```suggestion
    margin: theme.spacing(8, 'auto')
```
This could use the spacing method.